src/main/java/com/sap/orderpaymentgreen/util/rabbitMQ/Consumer.java

```diff
@@ -15,12 +16,16 @@
 public class Consumer {
     @Autowired
     private Producer defaultExchangeProducer;
+
+    @Autowired
+    private PaymentService paymentService;
     ObjectMapper objectMapper = new ObjectMapper();
     @RabbitListener(queues = {QUEUE_WAITING_FOR_CHARG})
     public void listenOnWaitingChargQueue(String order) throws JsonProcessingException {
         OrderDTO orderDTO = objectMapper.readValue(order, OrderDTO.class);
-         orderDTO.setOrderStatus(CREATED);
+        orderDTO.setOrderStatus(CREATED);
```

Comment:
Why did you need to put status? 
You should get Order with status charging from oder-service